internal/renderers/skia/opengl_surface.rs

```diff
@@ -384,6 +384,10 @@ impl OpenGLSurface {
 impl Drop for OpenGLSurface {
     fn drop(&mut self) {
         // Make sure that the context is current before Skia calls glDelete***
-        self.ensure_context_current().expect("Skia OpenGL Renderer: Failed to make OpenGL context current before deleting graphics resources");
+        // In the event that this fails for some reason (lost GL context), convey that to Skia so that it doesn't try to call
+        // glDelete***
+        if self.ensure_context_current().is_err() {
```

Comment:
Shouldn't we somehow log this as an error?

Reply:
I'm not sure. We could print something to stderr, but the actual error (context loss) happened a lot earlier.

Reply:
Differently put: We could also always call abandon() - the resources will be deleted anyway by the context that we own. It's just us playing nice :)